### PR TITLE
Add chapter to migration guide on how to migrate to Volto

### PR DESCRIPTION
**Pull request by pbauer**

### CODE REVIEW

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Because I renamed this file to use dashes instead of underscores, and there were too many itty-bitty changes, I just pushed changes to this PR.

However I have several questions and comments that we need to resolve before I feel comfortable merging this PR. I wrote them inline as `todo`.

File: docs/backend/upgrading/upgrade-to-volto.md
Comment (stevepiercy) [inline docs/backend/upgrading/upgrade-to-volto.md]: I think that "upgrade" is less correct than "migration". In the instructions, we "Migrate RichText fields to Volto blocks" and the form's URI contains "migrate_to_volto". Let's use migrate instead of upgrade everywhere in this document, including its file name.

Admittedly it is a little bit weird because it is located within the "upgrading" directory, but I'm OK with a URI of `/backend/upgrading/migrate-to-volto.html`.

File: docs/backend/upgrading/upgrade-to-volto.md
Comment (stevepiercy) [inline docs/backend/upgrading/upgrade-to-volto.md]: Out of curiosity, what is the actual terminology used in the UI?

In my mind, one creates a Plone site with a frontend of either Volto or Classic UI. I dislike the phrase of "creating a Volto site" or "creating a Classic Plone site" as it implies that they are two totally different sites instead of just UIs. Both options create a "Plone site". Only the UI is the primary difference.

```suggestion
When creating a new Plone 6 site, you may choose between Volto and Classic UI for the frontend.
```

File: docs/backend/upgrading/upgrade-to-volto.md
Comment (stevepiercy) [inline docs/backend/upgrading/upgrade-to-volto.md]:
```suggestion
This choice is presented because there are some non-trivial differences between their configurations.
```

File: docs/backend/upgrading/upgrade-to-volto.md
Comment (stevepiercy) [inline docs/backend/upgrading/upgrade-to-volto.md]:
```suggestion
This document discusses these differences.
It also informs administrators and developers of how to migrate their existing Plone 6 site with Classic UI for its frontend to instead become compatible with Volto for its frontend.
```

File: docs/backend/upgrading/upgrade-to-volto.md
Comment (stevepiercy) [inline docs/backend/upgrading/upgrade-to-volto.md]:
````suggestion
```important
As a pre-requisite, your Plone site must be [upgraded to Plone 6](v60) before migrating to Volto for the frontend.
```
````

File: docs/backend/upgrading/upgrade-to-volto.md
Comment (stevepiercy) [inline docs/backend/upgrading/upgrade-to-volto.md]:
```suggestion
Plone provides a form `/@@migrate_to_volto` that allows you to run all the required changes to your existing Plone 6 site to make it compatible with Volto.
```

File: docs/backend/upgrading/upgrade-to-volto.md
Comment (stevepiercy) [inline docs/backend/upgrading/upgrade-to-volto.md]:
```suggestion
You can access this form in the browser when you are logged in as an administrator by opening `http://localhost:8080/Plone/@@migrate_to_volto` (where `localhost` is your server, `8080` is the port on which Plone is running, and `Plone` is the name of the Plone instance).
```

File: docs/backend/upgrading/upgrade-to-volto.md
Comment (stevepiercy) [inline docs/backend/upgrading/upgrade-to-volto.md]:
```suggestion
Additionally, after upgrading an existing site to Plone 6 (see {doc}`v60`) a message will appear, **You can prepare your site for Volto, the default frontend of Plone 6!**, with a link to that form.
```

File: docs/backend/upgrading/upgrade-to-volto.md
Comment (stevepiercy) [inline docs/backend/upgrading/upgrade-to-volto.md]:
```suggestion
A site that is made compatible with Volto will be accessible with Plone Classic UI, but it will behave differently.
```

File: docs/backend/upgrading/upgrade-to-volto.md
Comment (stevepiercy) [inline docs/backend/upgrading/upgrade-to-volto.md]:
```suggestion
For example, editors can only effectively work with the content using Volto because HTML is no longer editable in the TinyMCE editor used only in Classic UI.
```

File: docs/backend/upgrading/upgrade-to-volto.md
Comment (stevepiercy) [inline docs/backend/upgrading/upgrade-to-volto.md]: Thanks for the explanation, and that makes sense to me now. Please delete the todo discussion for this one.

File: docs/backend/upgrading/upgrade-to-volto.md
Comment (stevepiercy) [inline docs/backend/upgrading/upgrade-to-volto.md]:
```suggestion
If you are migrating an existing site to Plone 6 using [{py:mod}`collective.exportimport`](https://pypi.org/project/collective.exportimport) and want to use Volto in the new site, then you do not need to use the form `@@migrate_to_volto`.
```

File: docs/backend/upgrading/upgrade-to-volto.md
Comment (stevepiercy) [inline docs/backend/upgrading/upgrade-to-volto.md]: Merge with previous line.

```suggestion
```

File: docs/backend/upgrading/upgrade-to-volto.md
Comment (stevepiercy) [inline docs/backend/upgrading/upgrade-to-volto.md]:
```suggestion
All the changes documented above can be done efficiently during export and import.
[Read details](https://github.com/collective/collective.exportimport/issues/133).
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Mostly English grammar and syntax fixes, with a couple of questions and comments. I think this document should use "migrate" instead of "upgrade" throughout.

File: docs/backend/upgrading/upgrade-to-volto.md
Comment (pbauer) [reply docs/backend/upgrading/upgrade-to-volto.md]: Ok, I'm fine with using the term Migration for this :)

File: docs/backend/upgrading/upgrade-to-volto.md
Comment (pbauer) [reply docs/backend/upgrading/upgrade-to-volto.md]: I actually is "Create a new Plone site" and "Create Classic Plone site":

Bildschirmfoto 2022-05-17 um 19 06 53

I'll update accordingly

Comment (stevepiercy) [reply docs/backend/upgrading/upgrade-to-volto.md]: Where would be a good place to discuss this UI? I think it could be improved. I approach this with fresh eyes, and it is... confusing.

For this PR, aligning with the UI is the way forward.

File: docs/backend/upgrading/migrate-to-volto.md
Comment (mauritsvanrees) [inline docs/backend/upgrading/migrate-to-volto.md]: uses -> used

**Review by mauritsvanrees — DISMISSED** [review]
Very clear, thanks. 
One typo.

File: docs/backend/upgrading/upgrade-to-volto.md
Comment (stevepiercy) [reply docs/backend/upgrading/upgrade-to-volto.md]: Found it, and not easily.

https://github.com/plone/Products.CMFPlone/commit/cd57b6e82fca76a35caac018a18211e7a7dddc39

File: docs/backend/upgrading/migrate-to-volto.md
Comment (stevepiercy) [reply docs/backend/upgrading/migrate-to-volto.md]: Fixed in c1d559abe18885f39411b9b9dbb6577512f31111, along with other things to reduce some review cycles.

**Review by stevepiercy — APPROVED** [review]
I made several changes in c1d559abe18885f39411b9b9dbb6577512f31111. They should be reviewed by another pair of eyes.

**Comment by pbauer** [comment]
LGTM

**Pull request merged**
